mmdet/datasets/custom.py

```diff
@@ -88,8 +88,11 @@ def __init__(self,
                 self.proposal_file = osp.join(self.data_root,
                                               self.proposal_file)
         # load annotations (and proposals)
-        with self.file_client.get_local_path(self.ann_file) as local_path:
-            self.data_infos = self.load_annotations(local_path)
+        try:
+            with self.file_client.get_local_path(self.ann_file) as local_path:
+                self.data_infos = self.load_annotations(local_path)
+        except AttributeError:
+            raise AttributeError('Please upgrade mmcv to >= 1.3.16')
```

Comment:
How about we raise a warning here and simply change to use self.data_infos = self.load_annotations(self.ann_file)

Reply:
Also, need to handle line 98

Reply:
Done